scripts/ci/check_compliance.py

```diff
@@ -1319,54 +1469,54 @@ def check_no_undef_outside_kconfig(self, kconf):
         # zephyr-keep-sorted-start re(^\s+")
         "ALSO_MISSING",
         "APP_LINK_WITH_",
-        "APP_LOG_LEVEL", # Application log level is not detected correctly as
-                         # the option is defined using a template, so it can't
-                         # be grepped
+        "APP_LOG_LEVEL",  # Application log level is not detected correctly as
+        # the option is defined using a template, so it can't
+        # be grepped
```

Comment:
Formatting here new is a bit awkward, maybe if it's a multiline comment, it should be before the actual item

```python
        # Application log level is not detected correctly as
        # the option is defined using a template, so it can't
        # be grepped
        "APP_LOG_LEVEL", 
```